Assets/PatchKit Patcher/Editor/Building/PostBuild.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.CodeDom;
+using System.Linq;
+using JetBrains.Annotations;
+using UnityEditor;
+using UnityEditor.Callbacks;
+
+namespace PatchKit.Unity
+{
+    public class PostBuild
+    {
+        private static readonly BuildTarget[] InvalidBuildTargets = { BuildTarget.StandaloneOSXUniversal };
+
+        [PostProcessBuild, UsedImplicitly]
+        private static void Execute(BuildTarget buildTarget, string buildPath)
+        {
+            if (InvalidBuildTargets.Contains(buildTarget))
```

Comment:
I would change it to `ValidBuildTargets` instead of checking whether current one is invalid.

Reply:
I think it might be more difficult to manage in the long run - it will require more compilation guards depending on Unity version.

But if you insist, then sure, would this list of valid build targets be ok?
```
BuildTarget.StandaloneLinux, 
BuildTarget.StandaloneLinux64, 
BuildTarget.StandaloneWindows, 
BuildTarget.StandaloneWindows64, 
BuildTarget.StandaloneOSXIntel64
```

Reply:
> I think it might be more difficult to manage in the long run - it will require more compilation guards depending on Unity version.

Sure, but those BuildTargets are tested and supported by us, if Unity will introduce a new one, we would need to check it first, before marking it as valid.

> But if you insist, then sure, would this list of valid build targets be ok?

Sure that's fine!
Thanks!